docs/hubs-create-join-rooms.md

```diff
@@ -42,11 +42,11 @@ On your computer, click "Enter Room" and follow the prompts to select a username
 
 For troubleshooting information visit Hubs [troubleshooting](./hubs-troubleshooting) page.
 
-![Enter room on Wired in device](img/hubs-enter-connected-vr.jpeg)
+<!-- ![Enter room on Wired in device](img/hubs-enter-connected-vr.jpeg)
```

Comment:
Do you want to delete these comments since they're not needed anymore?

Reply:
I'm going to keep them as comments - we should re-enable this when we verify the desktop flow with the new UI. I haven't been able to do that yet so I'm going to keep it hidden for the initial launch